tools/python/convert_onnx_models_to_ort.py

```diff
@@ -3,21 +3,22 @@
 # Licensed under the MIT License.
 
 import argparse
-import glob
 import os
 import pathlib
-import re
 
 import onnxruntime as ort
 
 
-def _create_config_file_from_ort_models(model_path: pathlib.Path, enable_type_reduction: bool):
-    filename = 'required_operators_and_types.config' if enable_type_reduction else 'required_operators.config'
-    config_file_path = model_path.joinpath(filename)
+def _onnx_model_path_to_ort_model_path(onnx_model_path: pathlib.Path):
+    assert onnx_model_path.is_file() and onnx_model_path.match("*.onnx")
+    return onnx_model_path.with_suffix(".ort")
```

Comment:
pathlib.Path.match() is case sensitive on posix isn't it? Is so, shouldn't we allow any casing to match?

Reply:
right, it seems like the platform specific way - case insensitivity only on Windows. IMO this is expected behavior, let me know if you think otherwise.

---
In reply to: [595744972](https://github.com/microsoft/onnxruntime/pull/7019#discussion_r595744972) [](ancestors = 595744972)

Reply:
If I provide a directory as the input, why shouldn't it match a .ONNX file on linux? 

---
In reply to: [596285547](https://github.com/microsoft/onnxruntime/pull/7019#discussion_r596285547) [](ancestors = 596285547,595744972)

Reply:
i think the answer would be similar to that of the question of why file matching is case-sensitive on Linux and i don't know that answer. but Path.match() is consistent with the platform behavior

---
In reply to: [596564205](https://github.com/microsoft/onnxruntime/pull/7019#discussion_r596564205) [](ancestors = 596564205,596285547,595744972)

Reply:
updated to match case insensitively

---
In reply to: [597013795](https://github.com/microsoft/onnxruntime/pull/7019#discussion_r597013795) [](ancestors = 597013795,596564205,596285547,595744972)